### PR TITLE
Metrics

### PR DESCRIPTION
**Pull request by mpetrun5**
**Short description of work done**

Implemented metrics endpoint and added example dashboard

### PR Checklist

- [x] I have run linter localy
- [x] I have run unit and integration tests locally
- [x] Rebased to master branch / merged master
- [x] Updated CHANGELOG.md

### Changes

- added /metrics endpoint
- embedded app version on build
- added prometheus and grafana to docker-compose 
- added grafana example dashboard

Open to suggestions:

Screenshot from 2020-12-03 11-22-47
Screenshot from 2020-12-03 11-22-31
Screenshot from 2020-12-03 11-22-12

### Issues

Closes: #137

### CODE REVIEW

**Review by MakMuftic — DISMISSED** [review]
From a code perspective, everything seems good.

I would maybe just paint _Failed request_ and _Penalized nodes_ to red

File: Makefile
Comment (mpetrunic) [inline Makefile]: can you define flag as variable instead of copy pasting everywhere?

**Review by mpetrunic — COMMENTED** [review]
Let's not merge this until we merge payout milestone on master.

Could you update Readme.md with instructions installing prometheus(link to official docs), installing grafana (official docs + adding prometheus data source) + dashboard import?

Also document where can grafa be found when ran trough docker-compose.

**Comment by mpetrunic** [comment]
LGTM, but draft until we merge payout to master

File: docker-compose.yml
Comment (mpetrunic) [inline docker-compose.yml]: you could remove auth alltogether. Something related to anonymus access

**Review by mpetrunic — COMMENTED** [review]
add comment to readme.md#demo with grafana url.

**Review by MakMuftic — APPROVED** [review]
LGTM

**Review by mpetrunic — APPROVED** [review]

**Pull request merged**
